.buildkite/scripts/e2e-pipeline/bootstrap.py

```diff
@@ -160,9 +161,33 @@ def __reload_container(self) -> None:
                                 "Error occurred while reloading Logstash container, see logs for details.")
         time.sleep(20)  # give a time Logstash pipeline to fully start
 
+    def __is_version_gte(self, version: str, target_major: int, target_minor: int) -> bool:
+        # Remove -SNAPSHOT suffix if present
+        clean_version = version.replace("-SNAPSHOT", "")
+        parts = clean_version.split(".")
+        if len(parts) < 2:
+            return False
+        
+        try:
+            major = int(parts[0])
+            minor = int(parts[1])
+            if major > target_major:
+                return True
+            elif major == target_major:
+                return minor >= target_minor
+            else:
+                return False
+        except ValueError:
+            return False
+
     def __update_pipeline_config(self) -> None:
         local_config_file_path = ".buildkite/scripts/e2e-pipeline/config/"
-        config_file = "serverless_pipeline.conf" if self.project_type == "serverless" else "pipeline.conf"
+        if self.__is_version_gte(self.stack_version, 9, 3):
+            config_file = "pipeline-ea-without-ssl.conf"
```

Comment:
I cant think of a reason to special case this. I think we should figure out *why* this all the sudden does not use ssl rather than letting it use unencrypted.